http-client/src/test/groovy/io/micronaut/http/client/HttpHeadSpec.groovy

```diff
@@ -377,28 +377,28 @@ class HttpHeadSpec extends Specification {
 
         then:
         def ex = thrown(HttpClientResponseException)
-        ex.message == "Empty body"
+        ex.message == "Request [/head/date/3918-11-20] failed with error: Empty body"
 
         when:
         client.formatDateQuery(d)
 
         then:
         ex = thrown(HttpClientResponseException)
-        ex.message == "Empty body"
+        ex.message == "Request [/head/dateQuery] failed with error: Empty body"
 
-        when:
-        client.formatDateTime(dt)
-
-        then:
-        ex = thrown(HttpClientResponseException)
-        ex.message == "Empty body"
+//        when:
+//        client.formatDateTime(dt)
+//
+//        then:
+//        ex = thrown(HttpClientResponseException)
+//        ex.message == "Request [/head/dateTime/3918-11-20] failed with error: Empty body"
```

Comment:
This test should not be commented out

Reply:
Sorry, did that by mistake while debugging tests.